### PR TITLE
Permit users to accept invalid SSL certs if they'd like

### PR DESCRIPTION
**Pull request by r24y**
Useful on corporate intranets.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "httpntlm",
+  "name": "@r24y/httpntlm",
```
Comment (hsingh23) [inline package.json]: why did you change this?

Comment (r24y) [inline package.json]: I'm so sorry, I pushed my personal fork to npm under my username and didn't realize I'd kept the change in the branch I pushed for this PR. I'll correct this and push again.

**Comment by SamDecrock** [comment]
This was fixed with pull request #32
